<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelsPrintDialog</class>
 <widget class="QDialog" name="ChannelsPrintDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Print channels list</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/icons/icons.qrc">
    <normaloff>:/icons/apps/16/accessories-text-editor.png</normaloff>:/icons/apps/16/accessories-text-editor.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="channelListBox">
     <property name="title">
      <string>Print channels list</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="labelSelect">
        <property name="text">
         <string>Select information for printing:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkNum">
        <property name="text">
         <string>Channel number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkName">
        <property name="text">
         <string>Channel name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkType">
        <property name="text">
         <string>Channel type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkUrl">
        <property name="text">
         <string>URL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkCategories">
        <property name="text">
         <string>Categories</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkLanguage">
        <property name="text">
         <string>Language</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkEpg">
        <property name="text">
         <string>XMLTV ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkLogo">
        <property name="text">
         <string>Logo</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkNum</tabstop>
  <tabstop>checkName</tabstop>
  <tabstop>checkType</tabstop>
  <tabstop>checkUrl</tabstop>
  <tabstop>checkCategories</tabstop>
  <tabstop>checkLanguage</tabstop>
  <tabstop>checkEpg</tabstop>
  <tabstop>checkLogo</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../../resources/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
